<launch>

  <arg name="file"/>
  <arg name="interval" default="1"/>

  <node name="pcd_to_pointcloud" pkg="pcl_ros" type="pcd_to_pointcloud" args="$(env PWD)/$(arg file) $(arg interval)"/>

  <!-- launch tests -->
  <node name="pcl_tests_node" pkg="pcl_tests" type="pcl_tests_node" output="screen">
	<param name="cloud_topic" value="/cloud_pcd" />
  </node>

  <!-- <node name="rviz" pkg="rviz" type="rviz" args="-f base_link"/> -->

</launch>

